src/stylesheets/components/_concept.scss

```diff
@@ -1,3 +1,23 @@
+@keyframes capeFlow {
+
+  from {
+    d: path("M102.7,114.1c-19.8-8.3-41.8-12.4-61.3-3.2c-16,7.6-29.3,24.2-35.4,44.4c-1.9,6.4-3.2,13-4.4,19.6c-0.9,4.8-1.8,9.8-1.6,14.7c0.2,4.5,1.4,8.8,2.8,12.9c10.3,29.6,33.8,49.2,58.4,57.9s50.6,8,76.1,7.3c15.5-0.4,31.1-0.8,46.1-5.3c31.2-9.2,74.9-39.5,96.2-70.2c2.2-3.2,0.2-7.6-3.5-8H276c-17.4-1.6-47.9-15.2-58.1-33.1c-10.2-17.9-20.2-37.8-36.3-46.4c-6.2-3.3-13.1-4.7-19.8-5c-9.8-0.4-19.7,1.4-29.1,5.4");
+  }
+
+  to {
+    d: path("M105.2,114.1c-19.8-8.3-41.8-12.4-61.3-3.2c-16,7.6-36.9,25.9-43,46.1c-1.9,6.4,1.9,24.5,0.7,31.1c-0.9,4.8-1.8,9.8-1.6,14.7c0.2,4.5,1.4,8.8,2.8,12.9c10.3,29.6,36.2,36,60.8,44.7s50.6,8,76.1,7.3c15.5-0.4,31.1-0.8,46.1-5.3c31.2-9.2,48.8-9.2,70.1-39.9c2.2-3.2,0.2-7.6-3.5-8h-0.1c-17.4-1.6-21.8-45.5-32-63.4c-10.2-17.9-20.2-37.8-36.3-46.4c-6.2-3.3-13.1-4.7-19.8-5c-9.8-0.4-19.7,1.4-29.1,5.4");
+    fill: #173C4A;
+  }
+}
+
+.cape path {
+  animation: 3s ease-in-out infinite alternate capeFlow;
+}
+
+.cape {
+  transform: translate(20px, 18px);
```

Comment:
Why is the transform necessary here?

Reply:
When I pulled in the modified cape paths, they shifted up and left slightly. The transform addresses that issue - maybe it's a little hacky?